### PR TITLE
Install bootloader before kexec/reboot

### PR DESCRIPTION
**Pull request by nadvornik**

### CODE REVIEW

File: saltboot-formula/_states/saltboot.py [hunk 1]
```diff
@@ -1753,14 +1754,50 @@ def bootloader_updated(name, partitioning, images, terminal_kernel_parameters=No
...
+            res = __salt__['cmd.run_all']("mount -t proc proc " + prefix + "/proc")
+            if res['retcode'] > 0:
+                ret['comment'] += res['stdout'] + res['stderr']
+            res = __salt__['cmd.run_all']("mount -t sysfs sysfs " + prefix + "/sys")
+            if res['retcode'] > 0:
+                ret['comment'] += res['stdout'] + res['stderr']
+
+            res = __salt__['cmd.run_all']("chroot '" + prefix + "' sh -c 'mount -a ; pbl --install ; pbl --config ; test -f /boot/grub2/grub.cfg '", python_shell=True)
+            if res['retcode'] > 0:
+                ret['comment'] += 'Bootloader installation failed.\n' + res['stdout'] + res['stderr']
```
Comment (aaannz) [inline saltboot-formula/_states/saltboot.py:1795]: Can't this all be replaced by cmd.run_chroot? ( https://docs.saltproject.io/en/latest/ref/modules/all/salt.modules.cmdmod.html#salt.modules.cmdmod.run_chroot )

Comment (nadvornik) [reply saltboot-formula/_states/saltboot.py:1795]: probably yes,
it would change the minimal supported salt version to 3000.
I suppose it is ok.

Comment (aaannz) [reply saltboot-formula/_states/saltboot.py:1795]: I think only SLE11 has salt version < 3000 and for that we have separate saltboot. For the rest Uyuni already requires salt 3000 I think.

**Review by aaannz — APPROVED** [review]
LGTM

**Pull request merged**